LICENSE

```diff
@@ -208,6 +208,7 @@ limitations under the License.
    subcomponents is subject to the terms and conditions of the following
    licenses.
 
+    (ALv2 License) airflow.contrib.auth.backends.github_enterprise_auth
```

Comment:
It should be below with the other 3rd party licenses

Reply:
I wasn't sure on this one since it's "in" the Airflow module it seemed like it was a sub-component, not a third party library/dependency.

Will move.